spec/dummy/config/boot.rb

```diff
@@ -0,0 +1,5 @@
+# Set up gems listed in the Gemfile.
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../../../Gemfile', __FILE__)
+
+require 'bundler/setup' if File.exist?(ENV['BUNDLE_GEMFILE'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.